fem/ceed/interface/interface.hpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2010-2022, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#ifndef MFEM_LIBCEED_INTERFACE
+#define MFEM_LIBCEED_INTERFACE
+
+// Object wrapping a CeedOperator in a mfem::Operator.
+#include "operator.hpp"
+// Functions to initialize CeedBasis objects.
+#include "basis.hpp"
+// Functions to initialize CeedRestriction objects.
+#include "restriction.hpp"
+// Functions to initialize coefficients.
+#include "coefficient.hpp"
+// PA or MF Operator using libCEED.
+#include "integrator.hpp"
+// Utility functions
+#include "util.hpp"
+// Wrapper to include <ceed.h>
+#include "ceed.hpp"
+
+namespace mfem
+{
+
+namespace ceed
+{
+
+
+} // namespace ceed
+
+} // namespace mfem
```

Comment:
If these empty namespaces don't do anything, I think they can be deleted.

Reply:
done in [6e56564](https://github.com/mfem/mfem/pull/2952/commits/6e56564b2cde59391b469656653a8e50760903c4).